test/test.translator.js

```diff
@@ -0,0 +1,48 @@
+var program = require('../')
+  , should = require('should');
+
+var translations = {
+    'Usage:': 'Применение:',
```

Comment:
Thanks for adding the test!
Could you modify the indent width to 2?
Because the indentation width of other code is 2.